swagger.yaml

```diff
@@ -690,6 +690,12 @@ paths:
           type: string
           required: true
           description: the group's ID
+        - in: query
+          name: sort
+          description: The sort order of the returned set of users
```

Comment:
an example would be ideal 🙏

Reply:
Hmm, I don't like that comment (is it rendered?).

Given that [examples can only be singular](https://swagger.io/docs/specification/2-0/adding-examples/) then I suggest using an [enum](https://swagger.io/docs/specification/2-0/enums/) for the small number of options for this param

Reply:
e.g.
```yaml
   enum: [ "forename:asc", "forename:desc" ]
```
or whatever it is